README.md

```diff
@@ -173,6 +177,27 @@ Returns the contents of a given file.
 client.download_file("{file_id}", "{item_id}", "{vault_id}", "{content_path}")
 ```
 
+**Async way**
+
+All the examples above can work in async way. Here is an example:
+```python
+import asyncio
+
+# initialize async client by passing is_async = True
+client: Client = new_client(
+    "{1Password_Connect_Host}",
+    "{1Password_Connect_API_Token}",
+    True)
+
+async def main():
+    vaults = await client.get_vaults()
+    item = await client.get_item("{item_id}", "{vault_id}")
+    # do something with vaults and item
+    await async_client.session.aclose()  # close the client gracefully when you are done
```

Comment:
So if I understand correctly, it is recommended to do one of the following:
- Use the context-managed client to not think of closing the client at some point (maybe if looks better isolated if you do more interaction with the async client)
- Close the client (it can be anywhere in the code, but you should do it regardless)

If that is the case, then the example above is good as is. Thank you for the additional explanation. 😄